.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,16 +17,30 @@ Reviewing guidelines: https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/do
 
 ###### Things done
 
-<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->
+<!-- 
+Requirements: 
+- install `nixpkgs-review` for good with `nix-env -f '<nixpkgs>' -iA nixpkgs-review`
+- consult usage documentation: https://github.com/Mic92/nixpkgs-review#usage
+- setup github api token: https://github.com/Mic92/nixpkgs-review#github-api-token
+-->
+
+- [ ] Tested sucessful build of final PR `GITHUB_TOKEN=<YOUR_TOKEN> nixpkgs-review pr <pr-number>`.
```

Comment:
Also not sure it makes sense to mention `pr <pr-number>` in the checklist since when a person is contributing, there is no PR-yet.

For “built on platforms”, I can do that before submitting the PR, but here it forces me to leave the terminal to go edit the pull request after submitting it. I use [`gh`](https://github.com/cli/cli) so my current workflow is running `nix-build` and then checking the checkboxes in the text editor, before submitting.

Reply:
This looks like a good workflow. However:
- The pullrequesters (now) iare encuraged to complete the same worfklow as the reviewers
- That is, they are encouraged to `nixpkgs-review post-result` this is only available once the pr exists

This can't be addressed in this PR, but it might make sense to support creating a PR with `nixpkgs-review` altogether so that already the commit message can combine:
- PR template for the respective PR type (module update, package update, new, etc) with type-specific check boxes
- concatenated with the result of the nixpkgs-review evaluation

Reply:
https://github.com/Mic92/nixpkgs-review/issues/115

Reply:
Well, there is `nixpkgs-review rev HEAD` which can be used by the pullrequester, maybe we could add a comment like ``<!-- you can use `nixpkgs-review rev HEAD` to run the tests before PR is opened -->``

Reply:
Well, there is `nixpkgs-review rev HEAD` which can be used by the pullrequester, maybe we could add a comment like ``<!-- you can use `nixpkgs-review rev HEAD` to run the tests before PR is opened -->``